v2/concordium/types.proto

```diff
@@ -1306,6 +1357,63 @@ message DelegationEvent {
   }
 }
 
+//Token events originating from account transcations.
+message TokenEvent {
+  // A token transfer between two accounts.
+  message Transfer {
+    TokenAmount amount = 1;
+    AccountAddress from_address = 2;
+    AccountAddress to_address = 3;
+    optional string memo = 4;
+  }
+
+  oneof event {
+    Transfer transfer = 1;
+  }
+}
+
+//Token events originating from governance transactions.
+message TokenGovernanceEvent {
+  message Mint {
+    TokenId token_id = 1;
```

Comment:
I think we need some address for where these minted tokens appear

Reply:
Only the issuer can mint via a `governance-transaction` new tokens and currently they are minted always to the issuer balance (if we adhere to the written document). As such if we observe a PLT `Mint` event with its `token_id` we could look up the issuer of that token (so there is no need to add the address if we only allow minting to the issuer). Nonetheless, I think adding the address to the event gives us more flexibility in the future, to e.g. allow issuers to mint to any random account address. I would even suggest that we relax the document and allow issuers to mint to any account address (needs update to the written document).

Reply:
@DOBEN But this would have to be authorized somehow. Otherwise you might end up with funds on your account that you never wanted.

Reply:
Yes, I expected that you cannot prevent funds being minted or transferred to your account.

How would this be prevented/authorized if the issuer first mints funds to its own address and then transfers it to your account?

Reply:
>  if we observe a PLT Mint event with its token_id we could look up the issuer of that token
Yes, but it would be more useful for user to know which account and the node would have it ready when this event was produced anyway. I think these events in general should describe the change in state, and we should keep the amount of implicit information to a minimum.

Reply:
@DOBEN I think that would be the same logic as for the allow/deny lists. But I'm not convinced this is a feature we should have in the MVP for the minting/burning of tokens.